.github/workflows/ci.yml

```
@@ -0,0 +1,73 @@
+  pull_request:
+    branches: [develop, main]
+  push:
+    branches: [develop]
+
+# 같은 PR 에 연속으로 push 하면 이전 실행은 취소한다.
+concurrency:
+  group: ci-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  test:
```

actionlint (1.7.12)

[error] "steps" section is missing in job "test"

(syntax-check)